app/src/main/java/com/nextcloud/talk/diagnose/DiagnoseContentComposable.kt

```diff
@@ -59,6 +95,124 @@ fun DiagnoseContentComposable(data: State<List<DiagnoseActivity.DiagnoseElement>
                 }
             }
         }
+        ShowTestPushButton(onTestPushClick)
```

Comment:
Do not show the "Test push notifications" button when gplay services are not available
You can see the check for this in the diagnosis screen itself.
(So if users are asked to click the self test push and don't find it in diagnosis screen, we already know the problem is on device...)

If notification app is not installed it's fine for me to show the button. In this case the testresult will be 404 which is okay as long there is no "success" string shown (see other comment).